### PR TITLE
[v2] Update using-jss example

### PR DESCRIPTION
**Pull request by ajayns**
Ref: https://github.com/gatsbyjs/gatsby/issues/5598

### CODE REVIEW

**Comment by ajayns** [comment]
@m-allanson Please review!

File: examples/using-jss/package.json [hunk 1]
```diff
@@ -2,11 +2,14 @@
   "name": "gatsby-example-using-jss",
   "private": true,
   "description": "Gatsby example site using the JSS plugin",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
Comment (nihgwu) [inline examples/using-jss/package.json]: do we really want to bump this version?

Comment (ajayns) [reply examples/using-jss/package.json]: I took the other PRs as reference. https://github.com/gatsbyjs/gatsby/pull/5717 Check this out.

Comment (m-allanson) [reply examples/using-jss/package.json]: Hmm I don't think it really make sense for the examples to have versions at all, as they're not published anywhere. Some actually don't have a version, but most have stayed as v1. I've reverted this, but maybe we need to take a closer look at the example versions later.

Comment (ajayns) [reply examples/using-jss/package.json]: True, I agree, I was assuming, we were following the convention will shifting to Gatsby v2 and so went forward with the changes. And so yes, later we should have a look at all the examples versions.

**Pull request merged**